Keymaps/Default (Windows).sublime-keymap

```diff
@@ -62,4 +62,7 @@
 
     // Symbol Hover
     // {"keys": ["UNBOUND"], "command": "lsp_hover", "context": [{"key": "setting.lsp_active"}]},
+
+    // Override native save to handle Code-Actions-On-Save
+    { "keys": ["ctrl+s"], "command": "lsp_save", "context": [{ "key": "setting.lsp_active"}] },
```

Comment:
It should also have code actions present in the view settings.

```
{"key": "setting.lsp_code_actions_on_save"}
```

Reply:
Yeah, can probably do that. Whether it's it would really make a difference depends on whether we'll keep the default to be an empty dict and what does empty dict evaluates to (True or False) in this context (I suppose it should work like Python so `{}` should be "Falsey" but will have to check).

Reply:
- This check would not evaluate to True when `lsp_code_actions_on_save` is set in LSP settings
- Even when having it in global preferences, it would still evaluate to False regardless if `lsp_code_actions_on_save` Dict is empty or not. I guess those condition only handle booleans.

Reply:
I learned that my above suggestion only works for boolean settings, not dictionaries.